services/widestage.js

```diff
@@ -0,0 +1,35 @@
+var request = require('request');
+var CryptoJS = require("crypto-js");
+
+var domain = "http://localhost"
```

Comment:
@gilgad we need to consolidate all the configuration related items for widestage.... similar to the `MONGO_URI` and any other external resource we should rely on environment variables.

Reply:
This was an oversight. I will abstract all configuration to be environment variables.